app/views/splash_screen/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="splash-container">
+  <div class="header-splash"></div>
+
+  <div class="content-splash">
+  <h1 class="title-splash">Welcome to Budget App Rod!</h1>
+
+  <div class="links-container-splash">
+    <%= link_to 'Sign Up', new_user_registration_path, class: 'sign-btn' %>
+    <%= link_to 'Log In', new_user_session_path, class: 'log-btn' %>
+  </div>
+  </div>
+</div>
```

Comment:
- [ ] Please add the correct placement of the elements of the splash screen page and stick to the layout and colors of the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://github.com/rodrigopenayo98/Budget-app-rod/assets/68796232/368ece63-ce7f-4501-8594-334920da5d1e) | ![image](https://github.com/rodrigopenayo98/Budget-app-rod/assets/68796232/9155245e-3caa-4cb0-82cd-d19009a973c6)  |

Reply:
Done! ✅